content/swagger/resources/product_models/definitions/product_model.yaml

```diff
@@ -125,10 +125,10 @@ properties:
         enum: ['read_only','draft_in_progress','proposal_waiting_for_approval','working_copy']
   quality_scores:
     type: object
-    description: Product model quality scores for each channel/locale combination (<strong>only available on SaaS platforms</strong> and when the "with_quality_scores" query parameter is set to "true")
+    description: Product model quality scores for each channel/locale combination (<strong>only available since 7.0</strong> and when the "with_quality_scores" query parameter is set to "true")
```

Comment:
```suggestion
    description: Product model quality scores for each channel/locale combination (<strong>only available since the 7.0 version</strong> and when the "with_quality_scores" query parameter is set to "true")
```